core/Cargo.toml

```diff
@@ -16,7 +16,11 @@ path = "lib.rs"
 [features]
 default = ["v8_use_custom_libcxx"]
 v8_use_custom_libcxx = ["v8/use_custom_libcxx"]
-include_js_files_for_snapshotting = []
+# Enable to exclude JS sources from the binary (e.g. if they are already snapshotted).
+exclude_js_sources = []
+# Read JS sources at 'runtime' instead of statically including them. Only to be used
+# in build and dev scripts. Overrides `exclude_js_sources`.
+runtime_js_sources = ["exclude_js_sources"]
```

Comment:
This is already an improvement that there are docstrings here, but while you're at it, could you improve them to be more descriptive? Just reading them doesn't give me a good idea which one should be used in snapshotting. (`Only to be used build and dev scripts` is somewhat ambiguous - I don't understand what it means)